no-cft-envelopes-processed-alert.tf

```diff
@@ -0,0 +1,24 @@
+module "no-cft-envelopes-processed-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "No_cft_envelopes_processed_-_Blob_Router"
+  alert_desc = "Triggers when Blob Router did not process single CFT envelope in last hour within SLA."
+
+  app_insights_query = <<EOF
+traces
+| where message startswith "No Envelopes created in CRIME"
+| where timestamp > now(-70m) and timestamp  < now()
```

Comment:
do we need to include = `(timestamp  <= now())` ?

Reply:
this should purely be used by `frequency_in_minutes` and `time_window_in_minutes` parameters ;). so only latter should be 70 to reflect what query does here